### PR TITLE
Add multi-check rules

### PR DESCRIPTION
**Pull request by Honny1**
This PR adds multi-check rules to the HTML report. 

Example:
image

Depend on: #140

### CODE REVIEW

File: openscap_report/scap_results_parser/oval_and_cpe_tree_builder.py [hunk 1]
```diff
@@ -9,55 +9,89 @@
...
+        )
+        if len(source_id) == 1:
+            self.cpe_source = source_id.pop()
+
+    def _get_dict_of_oval_cpe_definitions(self):
+        if self.cpe_source in self.reports_with_oval_definitions:
+            return self.reports_with_oval_definitions[self.cpe_source]
+        logging.warning((
+            "The given input does not contain a clear mapping of the OVAL definition used"
+            " for CPE checks. The results of the OVAL definition in the CPE checks could "
```
Comment (evgenyz) [inline openscap_report/scap_results_parser/oval_and_cpe_tree_builder.py:55]: Spaces: at the end or at the start?

File: openscap_report/scap_results_parser/scap_results_parser.py [hunk 2]
```diff
@@ -107,4 +107,11 @@ def parse_report(self):
         self._debug_show_rules(rules)
         report.rules = rules
         report.groups = groups
+
+        if len(report.profile_info.selected_rules_ids) > 0:
+            report.profile_info.selected_rules_ids.extend(rule_parser.to_select_rule_ids)
```
Comment (evgenyz) [inline openscap_report/scap_results_parser/scap_results_parser.py:112]: That should be a `ProfileInfo` mutate method, like `extend_selected_rules_ids`. Along with the logic how/when to do that `if len(report.profile_info.selected_rules_ids) > 0:`.

Why do you update it only when it is not empty by the way?

**Review by evgenyz — CHANGES_REQUESTED** [review]

File: openscap_report/scap_results_parser/scap_results_parser.py (hunk 2 above)
Comment (Honny1) [reply openscap_report/scap_results_parser/scap_results_parser.py:105]: There are two approaches how to determine if the rule is selected. 
- First is to get information from elements `<xccdf-1.2:select ...>`. A list of `selected_rules_ids` represents these elements. 
- The second is to get information from the result of the rule. 

The second approach is used if select elements aren't present in the ARF report. That is why the list is not extended when the list is empty.

**Comment by Honny1** [comment]
Changes:
- Created ProfileInfo mutate methods for selecting and deselecting rules
- Consistently split strings (previous PR)

**Pull request merged**